WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -868,15 +866,9 @@ - (void)setupReachability
 {
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Warc-retain-cycles"
```

Comment:
This is not part of your PR but it caught my attention... feel like removing this in your PR?  See my comment below...